tools/tpch-tools/README.md

```diff
@@ -0,0 +1,34 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## Usage
+
+These scripts are used to make tpc-h test.
+follow the steps below:
+
+### 1. build tpc-h dbgen tool.
+    ./build-tpch-dbgen.sh
+### 2. generate tpc-h data. use -h for more infomations.
+    ./gen-tpch-data.sh -s 1
+### 3. create tpc-h tables. modify `doris-cluster.conf` to specify doris info, then run script below.
+    ./create-tpch-tables.sh
+### 4. load tpc-h data. use -h for help.
+    ./load-tpch-data.sh
+### 5. run tpc-h queries.
+    ./run-tpch-queries.sh
```

Comment:
In fact, the test set query of tpch is slightly different under different data volumes.
So it's best to explain in the document that the query we give is a query under how big a dataset.